llvm/lib/Target/RISCV/MCTargetDesc/RISCVMCCodeEmitter.cpp

```diff
@@ -116,39 +116,68 @@ void RISCVMCCodeEmitter::expandFunctionCall(const MCInst &MI, raw_ostream &OS,
   MCInst TmpInst;
   MCOperand Func;
   MCRegister Ra;
+  bool IsCap;
```

Comment:
Same is true for Ra, and either true for Func or it has a default constructor so the assert would fire. I can see the argument for the default init wrt future merges, but it would look a bit out of place...

Reply:
New entries here should presumably give merge conflicts though so won't silently use uninitialised data?